core/modules/qdisp/testQDisp.cc

```diff
@@ -266,23 +266,11 @@ BOOST_AUTO_TEST_CASE(QueryResource) {
     r->ProvisionDone(NULL);
     BOOST_CHECK(status.getInfo().state  == qdisp::ExecStatus::PROVISION_NACK);
     char buf[20];
-    strcpy(buf, qdisp::XrdSsiSessionMock::getMockString(false));
-    qdisp::XrdSsiSessionMock xsMockFalse(buf);
-    r = new qdisp::QueryResource(s.resource.path(), s.request, s.requester,
-            qdisp::Executive::newNotifier(ex, refNum),
-            retryTest, status);
-    r->ProvisionDone(&xsMockFalse);
-    BOOST_CHECK(status.getInfo().state  == qdisp::ExecStatus::REQUEST_ERROR);
-    BOOST_CHECK(retryTest->_retryCalled == true);
-    // At this point, r has been deleted by QueryResource::ProvisionDone
-
-    LOGF_INFO("QueryResource test 2");
-    retryTest->_retryCalled = false;
+    strcpy(buf, qdisp::XrdSsiSessionMock::getMockString(true));
```

Comment:
I wasn't super clear on how the test plumbing worked downstream of here -- that is, I didn't want to remove functionality in the mocks that you think might be used in other tests?  On your recommend, happy to remove?